pretix_sepadebit/signals.py

```diff
@@ -59,6 +68,70 @@ def register_csv(sender, **kwargs):
     return DebitList
 
 
+@receiver(register_mail_placeholders, dispatch_uid="payment_sepadebit_placeholders")
+def register_mail_renderers(sender, **kwargs):
+
+    ph = [SimpleFunctionalMailTextPlaceholder(
+          'due_date', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.due.date, sample=date.today()
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'account', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.info_data.get('account', " "), sample="Max Mustermann"
```

Comment:
``account_holder`` is probably clearer, after all "Max Mustermann" isn't "the account"